roles/mariadb/defaults/main.yml

```diff
@@ -1,3 +1,3 @@
 keyserver_fingerprint: "0xcbcb082a1bb943db"
-mirror: nyc2.mirrors.digitalocean.com
+mirror: "ftp.nluug.nl/db"
```

Comment:
They had some hickups: https://status.digitalocean.com/

Def change it back though.